src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -947,19 +947,15 @@ const StorageRow = ({ connectionName, allowNoDisk, storageSize, storageSizeUnit,
         <SelectOption key="NewVolumeRAW" value="NewVolumeRAW">{_("Create new raw volume")}</SelectOption>
     ];
     if (allowNoDisk) {
-        StorageSelectOptions.push(<>
-            <Divider key="dividerNoStorage" />
-            <SelectOption value="NoStorage" key="NoStorage">{_("No storage")}</SelectOption>
-        </>);
+        StorageSelectOptions.push(<Divider key="dividerNoStorage" />);
+        StorageSelectOptions.push(<SelectOption value="NoStorage" key="NoStorage">{_("No storage")}</SelectOption>);
```

Comment:
The value is seen in the screen shot that seems wrong.

Reply:
Hm, I don't understand how that could have changed, nor can I reproduce it locally, but it did fail again after a retry. :exploding_head:

Reply:
I have a plausible theory, added a separate commit to fix the race.

Reply:
Haha no, we both [read this wrong](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1274-20231030-135040-a4e4a274-fedora-38/pixeldiff.html#TestMachinesCreate-testCreateBasicValidation-auto) -- it's the *current main* screenshot which is bogus! That's the one that said "NoStorage".

Reply:
Woops! Nice, we are fixing real bugs \o/